### PR TITLE
Added readme file with eslint fixes

### PR DESCRIPTION
**Pull request by sanjayadesilva123321**

### CODE REVIEW

**Review by mewancs — APPROVED** [review]

**Comment by ThiwankaB** [comment]
Commit message rules and linting might not be executed during commits. Could you please double-check to ensure that the functionality is working correctly? @sanjayadesilva123321

**Review by ThiwankaB — CHANGES_REQUESTED** [review]
Commit message rules and linting might not be executed during commits. Could you please double-check to ensure that the functionality is working correctly?

**Comment by sanjayadesilva123321** [comment]
> Commit message rules and linting might not be executed during commits. Could you please double-check to ensure that the functionality is working correctly?

Updated

**Review by ThiwankaB — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -81,3 +81,6 @@ PORT=4200
 
 REACT_APP_BACKEND_API_BASE_URL will use for the backend api base url.
 PORT variable determines the port on which the frontend web application runs.
+
+## React version
```
Comment (ThiwankaB) [inline README.md:85]: This should be the "NodeJS" version

**Comment by ThiwankaB** [comment]
If you make alterations after receiving approval on the pull request, please refrain from merging directly. Instead, kindly initiate the review process again, considering the new modifications you've implemented.

**Pull request merged**